buildSrc/src/main/java/com/uber/okbuck/core/model/android/Keystore.java

```diff
@@ -0,0 +1,19 @@
+package com.uber.okbuck.core.model.android;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class Keystore {
+
+  static Keystore create(String storeFile, String alias, String storePassword, String keyPassword) {
+    return new AutoValue_Keystore(storeFile, alias, storePassword, keyPassword);
+  }
+
+  public abstract String getStoreFile();
+
+  public abstract String getAlias();
```

Comment:
probably should have been named `getKeyAlias`

Reply:
Feel free to open a PR :)

Reply:
https://github.com/uber/okbuck/pull/722